contrib/podmanremoteimage/Containerfile

```diff
@@ -0,0 +1,10 @@
+FROM registry.access.redhat.com/ubi8/go-toolset:latest AS builder
```

Comment:
All of our other container images are based on `registry.fedoraproject.org/fedora:latest`, for consistency it would be good to use that here too.  An alternative could be `registry.fedoraproject.org/fedora-minimal:latest`.  In either case, check out the `contrib/podmanimage/*/Containerfiles` to see how we install `podman`.  It's _MUCH_ faster to install than to compile during image-build time (due to needing emulation).

Reply:
errr...nevermind :disappointed: that won't work for Mac or Windows will it?  My concern is the multi-arch compiles are extremely slow under emulation.  Hmmmm.  Perhaps the thing to do is just grab the binaries directly from the CI system's `artifacts` task.  They're not signed, but will be posted continuously for `main`, and recent release-branches (where we added ARM64 builds).

Reply:
@cevich Does it also published for specific tags like rc one? One of the use case which I am trying to solve with it, if we do any release on github (with specific branch) be it alpha, beta, rc we should have release binaries for those bits (longer term) either through the container image or from the CI artifacts. If I see https://github.com/containers/podman/pull/15202 (which is the PR for rc3) then looks like we just use the rc3 commit to do the tagging and there is no CI artifact for it.

Reply:
Mostly, I don't think tags to, but for sure on most recent branches (and going forward) the CI system continuously publishes artifacts at a consistent URL.  The URL will be of the form specified here: https://cirrus-ci.org/guide/writing-tasks/#latest-build-artifacts